package.json

```diff
@@ -24,5 +24,8 @@
   "gitHead": "dcde154040d17d321664a7efc38e8531330335f9",
   "devDependencies": {
     "tap": "~0.4.8"
+  },
+  "dependencies": {
+    "emoji-regex": "^6.1.0"
```

Comment:
Please use `~` as range specifier here.
I'm conservative with deps and also this avoids breaking older npms which might be the reason the build is breaking on travis right now.